pkg/controller/clusterserviceversion/clusterserviceversion_controller.go

```diff
@@ -66,7 +74,11 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 			UpdateFunc: func(evt event.UpdateEvent) bool {
 				_, okAllNamespace := evt.MetaNew.GetLabels()[allnamespaceTag]
 				watchLabel, watchOk := evt.MetaNew.GetLabels()[watchTag]
-				_, ignoreOk := evt.MetaNew.GetAnnotations()[IgnoreTag]
+				_, ignoreOk := evt.MetaNew.GetAnnotations()[ignoreTag]
+
+				if strings.Contains(evt.MetaNew.GetName(), utils.CSV_NAME) {
```

Comment:
Need to update this to look for our meter definition annotation and queue if it's present.